TimberAPI/SpecificationSystem/Fixes/CustomSpecifications/Golems/GolemFactionSpecificationObjectDeserializer.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using Timberborn.Persistence;
+
+namespace TimberbornAPI.SpecificationSystem.Fixes.CustomSpecifications.Golems
+{
+    public class GolemFactionSpecificationObjectDeserializer : IObjectSerializer<GolemFactionSpecification>
+    {
+        public void Serialize(GolemFactionSpecification value, IObjectSaver objectSaver)
+        {
+            throw new NotSupportedException();
```

Comment:
Would be a good idea to document why this is not supported

Reply:
// Not used for saving or something ?

Reply:
Yeah, and this was basically copy-paste from TB source? That could be added as well, so that we know the actual devs have it this way